databricks/koalas/base.py

```diff
@@ -179,9 +179,19 @@ def __sub__(self, other):
     __mul__ = _column_op(spark.Column.__mul__)
 
     def __truediv__(self, other):
+        """
+        __truediv__ has different behaviour between pandas and PySpark for several cases.
+        1. When divide np.inf by zero, PySpark returns null whereas pandas returns np.inf
+        2. When divide positive number by zero, PySpark returns null whereas pandas returns np.inf
+        3. When divide -np.inf by zero, PySpark returns null whereas pandas returns -np.inf
+        4. When divide negative number by zero, PySpark returns null whereas pandas returns -np.inf
+        """
```

Comment:
Can we have an ascii table to show the diff?

Reply:
Sure, I just added a table